app/src/main/java/org/jellyfin/mobile/bridge/ExternalPlayer.kt

```diff
@@ -42,6 +43,15 @@ class ExternalPlayer(private val fragment: WebViewFragment) : KoinComponent {
                     putExtra("position", mediaSource?.mediaStartMs)
                     putExtra("return_result", true)
                     putExtra("secure_uri", true)
+                    if (mediaSource?.subtitleTracksGroup?.selectedTrack != -1) {
+                        val selectedTrack = mediaSource?.subtitleTracksGroup?.tracks?.get(mediaSource?.subtitleTracksGroup?.selectedTrack as Int)?.url
+                        if (selectedTrack != null) {
```

Comment:
```suggestion
                    val selectedTrack = mediaSource?.subtitleTracksGroup?.tracks?.getOrNull(mediaSource?.subtitleTracksGroup?.selectedTrack as Int)?.url
                    if (selectedTrack != null) {
```

I think you can do something like this